install_config/topics/ocp_osp_admin_tasks.adoc

```diff
@@ -0,0 +1,65 @@
+[[osp_accounts]]
+==== Creating OpenStack User Accounts, Projects and Roles
+
+Before installing {product-title}, the Red Hat OpenStack Platform (RHOSP) environment requires a project (tenant)
```

Comment:
project, or tenant. that

Reply:
It's really not project or tenant, its more that people use the words project and tenant to mean the same thing. Would you want me to have it as product/tenant instead?

Reply:
@rlopez133 What we can do is, in the first instance, we can just say something like:

requires a project, often referred to as a _tenant_.  Then, we can just refer to it as a project in the rest of the docs. Thanks!